frontend/src/app/modules/shared/components/chart-switch-menu/chart-switch-menu.component.html

```diff
@@ -1,8 +1,13 @@
 <h1>
   <span *ngFor="let chart of chartSwitchMenu">
-    <span *ngIf="router.url.startsWith(chart.baseUrl); else showIcon">
-      {{ chart.label | translate }} (Dev)
+    <span *ngIf="router.url.startsWith(chart.baseUrl); else showDev">
+      {{ chart.label | translate }}
     </span>
+    <ng-template #showDev>
+      <span *ngIf="router.url.startsWith(chart.devUrl); else showIcon">
+        {{ chart.label | translate }} {{ router.url.includes('Dev/') ? '(Dev)' : '' }}
+      </span>
+    </ng-template>
     <ng-template #showIcon>
       <a [href]="chart.baseUrl + queryString" class="btn">
         <i [ngClass]="chart.icon"></i>
```

Comment:
Shouldn't the link in the icons also be distinguished between baseUrl and devUrl?

Reply:
I would say the navigation (in the navbar and in the chart switch menu) should only lead to working charts and not to the charts under development. Those should only be available via URL.

Reply:
My idea is, if I am on a dev page, that I can then switch to other dev pages via the Chart Switch menu, because it might make development easier.

Reply:
But it doesn't really matter either.